### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Call to a member function getFrameFilters() on null in vendor/filp/whoops/src/Whoops/Handler/PlainTextHandler.php:285

### ISSUE THREAD

**Issue by bener07** [filp/whoops#752]
After 1-day using whoops everything was okay but out of a sudden it crashed here is the full log of it:
```
PHP Fatal error:  Uncaught Error: Call to a member function getFrameFilters() on null in /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/filp/whoops/src/Whoops/Handler/PlainTextHandler.php:285
Stack trace:
#0 /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/filp/whoops/src/Whoops/Handler/PlainTextHandler.php(189): Whoops\Handler\PlainTextHandler->getTraceOutput()
#1 /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/filp/whoops/src/Whoops/Handler/PrettyPageHandler.php(292): Whoops\Handler\PlainTextHandler->generateResponse()
#2 /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/filp/whoops/src/Whoops/Run.php(398): Whoops\Handler\PrettyPageHandler->handle()
#3 /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(345): Whoops\Run->handleException()
#4 /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(324): Illuminate\Foundation\Exceptions\Handler->rend in /home/bernandre07/Desktop/dev/php/lanparty/website/vendor/filp/whoops/src/Whoops/Handler/PlainTextHandler.php on line 285
```

**Comment by AmraniCh** [comment]
Hey, thanks for opening this issue.
Until the bug is fixed and PR #751 got merged, try to revert to version **2.14.6**.

**Comment by bener07** [comment]
Thanks. it worked ;)
